### PR TITLE
Added js middleware docs and fixed #50

### LINKED ISSUE #50: collapsing of content
**As try dashboard should have some space between the other components**

**Screenshot:**
Screenshot from 2020-02-21 11-51-12

**Additional Context:**
If this really is a bug, I would like to take this up.

### PR DESCRIPTION
**Pull request by romitkarmakar**

### CODE REVIEW

**Comment by hsluoyz** [comment]
Too many things in one PR.

**Comment by nodece** [comment]
Hi @romitkarmakar, please create a new PR for each commit.

File: docs/Middlewares.md [hunk 1]
```diff
@@ -68,3 +68,142 @@ Name | Description
...
+- Create ```model.conf``` and ```policy.csv``` file in your project root folder, if not already created. For reference, check this link: https://github.com/node-casbin/express-authz/tree/master/examples
+- Put this code inside your app.js file
+```javascript
+const express = require('express')
+const app = express()
+
+app.use((req, res, next) => {
+  req.locals = req.locals || {}
+  const username = req.get('Authorization') || 'anonymous'
+  req.locals.currentUser = {username}
```
Comment (hsluoyz) [inline docs/Middlewares.md]: We don't need such details for this page.

Comment (romitkarmakar) [reply docs/Middlewares.md]: Ok I will not add such integration details here.

Comment (hsluoyz) [reply docs/Middlewares.md]: It should be in each repo's README.

Comment (nodece) [reply docs/Middlewares.md]: We can add the middleware repo to https://casbin.org/docs/en/middlewares.

Comment (romitkarmakar) [reply docs/Middlewares.md]: But, middleware repos for egg, express and koa are already included in this link https://casbin.org/docs/en/middlewares, only it doesn't have their integration examples in this page.

**Pull request closed without merging**